modules/luci-base/htdocs/luci-static/resources/form.js

```diff
@@ -4007,15 +4007,15 @@ var CBIFlagValue = CBIValue.extend(/** @lends LuCI.form.FlagValue.prototype */ {
 	 * Sets the input value to use for the checkbox checked state.
 	 *
 	 * @name LuCI.form.FlagValue.prototype#enabled
-	 * @type number
+	 * @type string
 	 * @default 1
```

Comment:
This change is not correct: since luci accepts 1 as true and 0 as false, whether quoted or not.

Reply:
the comment of a commit:


    luci-base: FlagValue fix type of enabled and disabled properties
    The form.Flag checkbox allows to specify a value for enabled/disabled e.g. for a dropbear:
    
        o = s.option(form.Flag, 'PasswordAuth');
        o.enabled  = 'on';
    
    So the property type should be a string but not just a number.


It's even marked by IntelliJ IDE as a mistyping.